google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -733,8 +737,12 @@ def get_access_policy(self, client=None):
             dict: Access Policy.
         """
         try:
-            data = client.iter_crm_org_access_policies(self['name'])
-            return data
+            access_policies = []
+            access_policy_iter = (
+                client.iter_crm_org_access_policies(self['name']))
+            for access_policy in access_policy_iter:
+                access_policies.append(access_policy)
+            return access_policies
```

Comment:
Previously the get policy methods were returning generators and this was not being written to the database correctly.